src/pages/ManageVaccineStockPage.js

```diff
@@ -196,12 +196,10 @@ export class ManageVaccineStockPage extends React.Component {
   // be available.
   componentDidMount = () => {
     const { database, initialLocation } = this.props;
-    const fridges = database.objects('Location').filter(location => location.isFridge);
-    fridges.unshift({ description: LOCALIZATION.misc.allLocations });
+    const fridges = database.objects('Fridge');
+    fridges.slice().unshift({ description: LOCALIZATION.misc.allLocations });
     this.LOCATION_FILTERS = fridges;
-    this.ITEMS = database
-      .objects('Item')
-      .filtered('category.name BEGINSWITH[c] $0 && category.name ENDSWITH[c] $0 ', 'vaccine');
+    this.ITEMS = database.objects('Item').filtered('category.name CONTAINS[c] $0 ', 'vaccine');
```

Comment:
should we add VaccineItem as an object (i.e. like Fridge) ?

Reply:
18bdf39e201e70828c47827044e83428f092bedd